.github/workflows/docker-images.yml

```diff
@@ -0,0 +1,51 @@
+name: Docker Image CI
+
+on:
+  schedule:
+  - cron: '37 2 * * 1'
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'
+        required: true
+        default: 'warning'
+        type: choice
+        options:
+        - info
+        - warning
+        - debug
+
+jobs:
+
+  docker-buildx:
+
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - uses: actions/checkout@v3
+
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v2
+      with:
+        install: true
+
+    - name: Login to DockerHub
+      uses: docker/login-action@v2
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+    - name: Inspect builder
+      run: |
+          echo "Name:      ${{ steps.buildx.outputs.name }}"
+          echo "Endpoint:  ${{ steps.buildx.outputs.endpoint }}"
+          echo "Status:    ${{ steps.buildx.outputs.status }}"
+          echo "Flags:     ${{ steps.buildx.outputs.flags }}"
+          echo "Platforms: ${{ steps.buildx.outputs.platforms }}"
+
+    - name: Build and push
+      run: |
+          cd docker/ubuntu18.04-cuda10.0/
+          docker build --push --tag kaldiasr/kaldi:gpu-latest --tag kaldiasr/kaldi:gpu-ubuntu18.04-cuda10.0 .
```

Comment:
Shall we put a date on the tag so that users can report bugs/issues on a specific tag?

Reply:
sounds reasonable, I'll look into it
Do you suggest something like 
```
 --tag kaldiasr/kaldi:gpu-latest --tag kaldiasr/kaldi:gpu-ubuntu18.04-cuda10.0 --tag kaldiasr/kaldi:gpu-ubuntu18.04-cuda10.0-$(date %s)
 ```
?

Reply:
Yes, something like that. The main motivation is to let developers distinguish which version users are using when they are reporting issues.

It is not clear what the `latest` version refers to as users can install the `latest` version at different times.

Reply:
yeah, makes sense to me

Reply:
I think we should also keep a `latest` tag that always points to the latest image that's built